pkg/util/interval/btree_based_interval_test.go

```diff
@@ -279,7 +277,6 @@ func checkWithLen(t *testing.T, tree *btree, l int) {
 }
 
 func check(t *testing.T, tree *btree) {
-	t.Logf("tree: %s", tree.describe())
 	if !tree.isLeafSameDepth(t) {
 		t.Error("Not all the leaves have the same depth as the tree height")
```

Comment:
Except that `tree.describe()` for some of the trees used in this test is a megabyte. I'll leave better reporting of errors as an exercise for the next person to need to debug this.